CIPs/cip-0040.md

```diff
@@ -453,6 +453,12 @@ Multiple candidate implementations exist for this functionality. Linked below is
 cryptographic design for use in the ODIS pOPRF service. Other implementation with the same interface
 may also be used to implement the following service design.
 
+Note that in the proposed interface, and in the function proposal below, only a single public key,
```

Comment:
Looks good to me! I think this clarifies that the new ODIS construction would only need 1 new set of keys, not that each domain needs a new set of a keys. 

nit: I think "shareded" (line 457) is supposed to be "sharded" or "shared"